### ISSUE TITLE
RenderController outTransition not influenced by _transition

### ISSUE THREAD

**Issue by feidens** [gadicc/meteor-famous-views#208]
A better way for supporting user defined transitions is adding the hide method
```
fview.view.hide transition, ->
        # Now that transition is complete, we can destroy the old template
        fview._previousShow.destroy()  if fview._previousShow

fview.view.show child_fview, transition, ->
        # If _transitionOnce was used, now we can restore the defaults
        if origTransitionData.modifierName
          console.log "restore " + origTransitionData.modifierName
          data = FView.transitionModifiers[origTransitionData.modifierName]
          for key of data
            fview.view[key] data[key]
        fview._transition = origTransitionData.transition  if origTransitionData.transition
        return

      return
```

**Comment by gadicc** [comment]
Did you maybe want `_transitionModifier`?  That controls the in- and out- effects using the timing from the actual `_transition` (linear, spring, etc).  Otherwise if you're saying the in- functions are using transition correctly but the out- functions aren't, that could be a famous bug.  Can you provide a reproduction?  If all looks correct you might need to also do a pure famo.us reproduction, unfortunately.

**Comment by gadicc** [comment]
Hey, sorry, I replied to this before seeing your edit.  Did your changes help though?  RenderController's `show()` calls `hide()` anyways ([ref](https://github.com/Famous/famous/blob/v0.3.4/src/views/RenderController.js#L222-L229)), although I agree it's more syntactically correct to destroy the old instance on hide() vs the completion of show().  Also, are you using the latest version of famous-views?  I just recalled we did a `_transition` fix in the latest release, v0.1.31.
